site/src/pages/WorkspacePage/Workspace.tsx

```diff
@@ -220,12 +230,24 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
 
       <Margins css={styles.content}>
         <Stack direction="column" css={styles.firstColumnSpacer} spacing={4}>
-          {workspace.outdated && (
+          {!showAutostartBanner && workspace.outdated && (
             <Alert severity="info">
               <AlertTitle>An update is available for your workspace</AlertTitle>
               {updateMessage && <AlertDetail>{updateMessage}</AlertDetail>}
             </Alert>
           )}
+          {showAutostartBanner && (
+            <Alert severity="warning">
+              <AlertTitle>
+                Autostart has been disabled for your workspace.
+              </AlertTitle>
+              <AlertDetail>
+                A parameter mismatch has been detected between your workspace
+                and the active template version. Manually update your workspace
+                to reenable Autostart.
```

Comment:
Maybe "An update for your workspace is required before it can start, and the new version cannot be applied automatically." The user doesn't need to know what a "parameter" is, and it's a bit more general in case something else can trigger this later.